pkg/operation/botanist/health_check.go

```diff
@@ -555,7 +543,7 @@ func (b *Botanist) checkControlPlane(
 		return exitCondition, err
 	}
 	if gardenletfeatures.FeatureGate.Enabled(features.Logging) {
-		if exitCondition, err := checker.CheckLoggingControlPlane(b.Shoot.SeedNamespace, b.isLoggingHealthCheckObsolete(), condition, seedStatefulSetLister); err != nil || exitCondition != nil {
+		if exitCondition, err := checker.CheckLoggingControlPlane(b.Shoot.SeedNamespace, b.Shoot.GetPurpose() == gardencorev1beta1.ShootPurposeTesting, condition, seedStatefulSetLister); err != nil || exitCondition != nil {
 			return exitCondition, err
 		}
 	}
```

Comment:
A few lines below, there is a temporary code about the metrics server health checking that can be removed in g/g@v1.14.x.

Reply:
Yeah, I also saw it but let's keep it for few more releases to make the gardener updates smooth (if someone from the community is updating several minor versions at once.)